spirit/topic/notification/templates/spirit/topic/notification/index_unread.html

```diff
@@ -5,7 +5,7 @@
 {% block title %}{% trans "Unread notifications" %}{% endblock %}
 
 {% block content %}
-
+    <h1 class="headline"><a href="{% url 'spirit:topic:notification:index' %}">{% trans "Notifications" %}</a></h1>
```

Comment:
`h1` elements should not contain links. This should be a drop-down, like the one in `http://.../topic/private/`

Reply:
Well that's a personal taste. Since Spirit all use drop-down menu, let me see how to unify them.